app/models/json_builders/channels_json_builder.rb

```diff
@@ -19,16 +19,14 @@ class JsonBuilders::ChannelsJsonBuilder
   def initialize
     require "publishers/excluded_channels"
     @excluded_channel_ids = Publishers::ExcludedChannels.brave_publisher_id_list
-    @verified_channels = Channel.verified
```

Comment:
Question, is this being used in any views that you know of? if not then this is fine to delete.